python/lsst/ts/ess/csc/config_schema.py

```diff
@@ -49,29 +49,33 @@
       channels: 4
       device_type: FTDI
       ftdi_id: AL05OBVR
+      location: Test
     items:
       type: object
       properties:
         name:
-          description: Name of the sensor
+          description: Name of the sensor.
           type: string
         sensor_type:
-          description: Type of the sensor
+          description: Type of the sensor.
           type: string
           enum:
           - HX85A
           - HX85BA
           - Temperature
           - Wind
         channels:
-          description: Number of channels
+          description: Number of channels.
           type: integer
         device_type:
-          description: Type of the device
+          description: Type of the device.
           type: string
           enum:
           - FTDI
           - Serial
+        location:
+          description: The location of the device.
```

Comment:
Please add something about how to handle multi-channel devices (e.g. comma-separated string).

(I'm sure you know, but you can start a multi-line string with >-, then put the text indented on the following lines)

 I suppose you could also make the schema flexible enough to allow location to be a single string or a list of strings. That avoids the question of a separator and neatly avoids long description strings in yaml files. But it makes the configuration a bit harder to use and of course you can write long strings on multiple lines using `>-` as noted earlier in this command.

Reply:
Done.